src/plugins/dashboard/public/dashboard_app/top_nav/managed_popover.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+  EuiPopover,
+  EuiPopoverFooter,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import { DashboardAPI } from '../..';
+import { DashboardRedirect } from '../../dashboard_container/types';
+import { dashboardManagedBadge } from '../_dashboard_app_strings';
+
+interface ManagedPopoverProps {
+  dashboard: DashboardAPI;
+  redirectTo: DashboardRedirect;
+}
+
+export const ManagedPopover = ({ dashboard, redirectTo }: ManagedPopoverProps) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const text = dashboardManagedBadge.getText();
+  const buttonText = i18n.translate('dashboard.managedContentPopoverFooterText', {
+    defaultMessage: 'Duplicate this dashboard',
+  });
+  const cancelButtonText = i18n.translate('dashboard.managedContentPopoverFooterCancelText', {
+    defaultMessage: 'Cancel',
+  });
+
+  const button = (
+    <EuiButton
+      color="primary"
+      iconType="glasses"
+      fill
+      size="s"
+      aria-label={i18n.translate('dashboard.managedContentBadge.text', {
+        defaultMessage: 'Managed',
+      })}
+      onClick={() => {
+        setIsPopoverOpen(!isPopoverOpen);
+      }}
+    >
+      <EuiText size="s">
+        {i18n.translate('dashboard.managedContentBadge.text', {
+          defaultMessage: 'Managed',
+        })}
+      </EuiText>
+    </EuiButton>
```

Comment:
The `managed-content-badge` is used in **more** than just Dashboard, so I think it is important that we keep the dependency on `kbn-content-badge` in the Dashboard plugin - that way, if anything changes with this component, Dashboard's `managed` badge will remain consistent with those across Kibana. 

I think we could get away with getting the props from `getManagedContentBadge` and spreading them on an `EuiBadge` as our popover button, instead. This might require some type shenanigans, but the goal is consistency here - if something is used across Kibana and we remake it like this, this will inevitably cause drift between the managed badge in Dashboard and the ones used elsewhere. 

As another option (I haven't explored this one as deeply), we might be able to get away with not having a separate `ManagedPopover` component **at all** if we use `EuiWrappingPopover` - this should allow us to use the normal `kbn-content-badge` and add the props we need (`onClick`, etc) - then, using `EuiWrappingPopover`, we can target that badge via CSS selector in order to attach the popover to it. I think this **might** actually end up being a cleaner solution than what I suggested above (assuming we can get it to work), so I would be interested to see you try this **first** and see where it gets you! :)